doc/changelogs/CHANGELOG_V11.md

```diff
@@ -40,6 +41,188 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="11.13.0"></a>
+## 2019-03-28, Version 11.13.0 (Current), @targos
+
+### Notable Changes
+
+* **events**
+  * Added a `once` function to use `EventEmitter` with promises (Matteo Collina) [26078](https://github.com/nodejs/node/pull/26078).
+* **tty**
+  * Added a `hasColors` method to `WriteStream` (Ruben Bridgewater) [26247](https://github.com/nodejs/node/pull/26247).
+  * Added NO\_COLOR and FORCE\_COLOR support (Ruben Bridgewater) [26485](https://github.com/nodejs/node/pull/26485).
+* **v8**
+  * Added `v8.getHeapSnapshot` and `v8.writeHeapSnapshot` to generate snapshots in the format used by tools such as Chrome DevTools (James M Snell) [26501](https://github.com/nodejs/node/pull/26501).
+* **meta**
+  * [Gireesh Punathil](https://github.com/gireeshpunathil) is now a member of the Technical Steering Committee [26657](https://github.com/nodejs/node/pull/26657).
+  * Added [ZYSzys](https://github.com/ZYSzys) to collaborators [26730](https://github.com/nodejs/node/pull/26730).
```

Comment:
Shouldn't we use the actual name `Yongsheng Zhang` here? Ping @ZYSzys

Reply:
Yeah, that's my mistake. I copied the name from GH profile intsead of the readme

Reply:
Thanks for pointing this out.😄